core/src/components/radio-group/radio-group.ionic.scss

```
@@ -0,0 +1,38 @@
+@use "../../themes/ionic/ionic.globals" as globals;
+@import "./radio-group.common";
+
+// Ionic Radio Group Top
+// --------------------------------------------------
+
+.radio-group-top {
+  @include globals.typography(globals.$ion-body-md-regular);
+
+  margin-bottom: globals.$ion-space-100;
```

This seems like it's `space-400` when looking at the design. Please confirm.

```suggestion
  margin-bottom: globals.$ion-space-400;
```

---

You're correct. I'm not sure where I got 100 from. Updated: https://github.com/ionic-team/ionic-framework/pull/30271/commits/cf4bdfb9ba76c751c3d9b27d176c046ee39f71e3